client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -23,3 +30,8 @@ export const setSiteTitle = ( siteTitle: string ) => ( {
 	type: ActionType.SET_SITE_TITLE as const,
 	siteTitle,
 } );
+
+export const togglePageLayout = ( pageLayout: Template ) => ( {
```

Comment:
I don't like `toggle` as an action and prefer `add`/`remove`, but this is fine for now 🙂